### DISCUSSION TITLE
Farm Status Dashboard

### DISCUSSION THREAD

**Discussion by Polysemouse** [sunflower-land/sunflower-land#1095] [Ideas]
Background
---
There are more and more farmable resources that are added to Sunflower Land over time (eg. Crops, Minerals, More animals etc.).  It will be difficult for users to keep track of their resource collecting times and not missing them when these resources are ready for harvest.

Dashboard
---
**Purpose**

I propose adding a Farm Status Dashboard to Sunflower Land.  Users will be able to use the dashboard to quickly glance through their farm resources.  They can quickly see which resources are available to harvest, or how many hours/minutes until users can harvest their resources.

**Accessing the dashboard**

The dashbard will either be accessible by clicking the "Home" building, clicking a new noticeboard in the farm, or clicking an extra button in the menu in the top left corner of the UI

**UI**

![image](https://user-images.githubusercontent.com/107602352/173997079-d94bd326-4175-4f10-963f-0254fcbc71e2.png)

The dashboard will be split into multiple tabs to account for the numerous resources that are going to be introduced into the game in the future.  The "Crops" tab will contain sunflower, potato, pumpkin, ..., wheat; the "Minerals" tab will contain wood, stone, iron, gold etc, the "Animals" tab will contain chicken etc, the "Misc." tab will contain wishing well etc.

Each row in the dashboard will show the fastest time that resource will be able for collection. Eg.  If stone 1, stone 2, stone 3 takes 2h9m, 2h24m and 2h5m to recover respectively, the "Stone" column will show the shortest time "2h 5m".  If any of the resources in that column is ready for harvest/collection, it will show "Ready" with a green color.  If the resource countdown is not available (eg. if the users do not have any chickens, or if the chickens are not fed with wheat), it will show "N/A" with a red color (like the Gold column in the screenshot.  Gold won't be non-available before land expansion, but the screenshot only serves as an UI example).  The colors have to be implemented in the right way to be colorblind friendly.

**Comment by NotKeen** [comment]
I have like 5-6 Countdowns on my smartphone running all day haha (also from Million on Mars). So, maybe this isn't a problem you need to solve in-game, since there is already a solution existing outside of the game.

Just my opinion of course. I'm sure it could be useful to someone. For me, it wouldn't be a high priority tho, sorry to say.

**Reply by celpower** [reply]
But in MoM you can go to your profile and see if something is ready to collect. It would be nice to have something like that in SFL. Just with chickens alone I forget to look up in the map to collect

**Comment by Polysemouse** [comment]
@NotKeen It might not be a necessary feature for everyone, but it would be a nice QoL feature as more and more resources are added to the game

**Comment by Aminbtdds** [comment]
Great idea😍 it is a good improvement on stats monitoring, but ofc it takes time for devs and not their periority, imo after land expansion maybe🤔

**Comment by celpower** [comment]
If the dashboard is too much to ask it would be nice to have at least some visual on screen that something is ready. Specially with land expansion coming. If things are in different lands it might be hard to go from one to another looking for everything.
